authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/helper/EntityInfoMapHelper.java

```diff
@@ -0,0 +1,140 @@
+package io.mosip.authentication.common.service.helper;
```

Comment:
Is there a real need for EntityInfoHelper and EntityInfoMapHelper? Can we not just have one utils for EntityInfoUtil and have both of this in the single class? whats the reason behind multiple class?

Reply:
Also why is it called Entity? Why create a class for methods from  IdInfoHelper.java?

Reply:
Created a new class  EntityInfoUtil and added methods of both EntityInfoHelper and EntityInfoMapHelper class.

Reply:
https://github.com/mosip/id-authentication/blob/d3e2535a486a56c6d7e80f58da2f7397cc8602e0/authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/util/EntityInfoUtil.java#L53

Based on method name getIdEntityInfoMap Created class name.
Created methods from IdInfoHelper to remove circular dependency issue.

Reply:
IDInfoHelper class calling IdInfoFetcherImpl.
IdInfoFetcherIml class calling otpManager.
otpManager calling notificationservice.
Finally notification service class calling IdInfoHelper. 
so to break circular dependency created new classes.